alternator/executor.cc

```diff
@@ -1611,6 +1616,9 @@ static future<executor::request_return_type> create_table_on_shard0(service::cli
             throw api_error::validation("LocalSecondaryIndexes must be an array.");
         }
         for (const rjson::value& l : lsi->GetArray()) {
+            if (!index_supported_by_config) {
+                co_return api_error::validation("LocalSecondaryIndex with tablets requires the rf_rack_valid_keyspaces option to be enabled.");
```

Comment:
Is this error message analogous to the one we print on CQL?
I doubt that any user will have any clue what this message means, but maybe we already have documentation about this...